gameon17s99-impl/pom.xml

```diff
@@ -51,6 +51,23 @@
                     <lagomService>true</lagomService>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <configuration>
+                    <skip>false</skip>
+                    <images>
+                        <image>
+                            <build>
+                                <entryPoint>
+                                    java -cp '/maven/*' -Dplay.crypto.secret="${APPLICATION_SECRET:-none}" -Dplay.akka.actor-system="${AKKA_ACTOR_SYSTEM_NAME:-gameonservice-v1}" -Dhttp.address="$GAMEONSERVICE_BIND_IP" -Dhttp.port="$GAMEONSERVICE_BIND_PORT" -Dakka.actor.provider=cluster -Dakka.remote.netty.tcp.hostname="$(eval "echo $AKKA_REMOTING_BIND_HOST")" -Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT" $(IFS=','; I=0; for NODE in $AKKA_SEED_NODES; do echo "-Dakka.cluster.seed-nodes.$I=akka.tcp://$AKKA_ACTOR_SYSTEM_NAME@$NODE"; I=$[$I+1]; done) -Dakka.io.dns.resolver=async-dns -Dakka.io.dns.async-dns.resolve-srv=true -Dakka.io.dns.async-dns.resolv-conf=on play.core.server.ProdServerStart
```

Comment:
This is using a command line for running in a cluster... its OK if we want to deploy everything as single-node clusters for consistency sake, but in that case, Jason discovered a bug in the command line syntax that we should incorporate into this: https://github.com/lagom/activator-lagom-java-chirper/pull/89

Alternatively, since this service is not really cluster-compatible at the moment, we could simplify it to run non-clustered.